docker/junit-rule/src/main/java/org/arquillian/cube/docker/junit/Reflections.java

```diff
@@ -0,0 +1,56 @@
+package org.arquillian.cube.docker.junit;
+
+
+import java.lang.reflect.Field;
+import java.lang.reflect.ParameterizedType;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Reflections {
+
+    public static final void injectObject(Object instance, Field field, Object injection) throws IllegalAccessException {
+
+        field.setAccessible(true);
+        field.set(instance, injection);
+
+    }
+
+    public static final List<Field> findAllFieldsOfType(Class<?> clazz, Class<?> fieldType, Predicate<Field> restriction) {
```

Comment:
same as above

Reply:
https://stackoverflow.com/questions/1743715/behaviour-of-final-static-method